### PR TITLE
UnicodeMath parsing rules

### LINKED ISSUE #31: Implement Formula to support UnicodeMath 3.2 conversion
The plurimath gem should re-implement the functionality of converting a Formula into Unicode math, which is a variant of textual math like HTML math.

Please refer to the code in the [unitsml gem](https://github.com/unitsml/unitsml).

### LINKED ISSUE #32: Implement parsing of Unicode math into Formula
The plurimath gem should implement the functionality of parsing Unicode math into a Formula:

Please refer to the code in the [html2asciimath](https://github.com/plurimath/html2asciimath) and the [unitsml](https://github.com/unitsml/unitsml) gems.

### PR DESCRIPTION
**Pull request by suleman-uzair**
This PR adds parsing rules for UnicodeMath math following **[UnicodeMath documentation](https://www.unicode.org/notes/tn28/UTN28-PlainTextMath-v3.2.pdf)**.

closes #31

### CODE REVIEW

File: lib/plurimath/math.rb [hunk 1]
```diff
@@ -14,6 +14,7 @@
 require_relative "math/formula"
 require_relative "math/function"
 require_relative "asciimath/parser"
+require_relative "unicode/parser"
```
Comment (ronaldtse) [inline lib/plurimath/math.rb:17]: Can we rename to “unicodemath” because it’s one word?

**Comment by ronaldtse** [comment]
Thank you @suleman-uzair !

**Comment by ronaldtse** [comment]
@suleman-uzair can you add a blog post about UnicodeMath support on plurimath.org? Thanks!

**Comment by suleman-uzair** [comment]
> @suleman-uzair can you add a blog post about UnicodeMath support on plurimath.org? Thanks!

@ronaldtse 
Sure. Right now, the transformation is not finished for **UnicodeMath**. I’ll add the blog post once I’m done with that in **Plurimath**.

**Pull request merged**